libs/language-server/src/grammar/main.langium

```diff
@@ -25,3 +26,5 @@ entry JayveeModel:
   )*;
 
 
+ImportDefinition:
+  'use' '*' 'from' path=STRING ';';
```

Comment:
This can later be refined to use named imports right?

Reply:
Yes! That's the plan :smile: